tabuckner/broskichat
# Home Page v1

### Project Changed


**- What I Did**
Added Home Page component w/ subcomponents. Added a temporary patch to header for layout checks when validating my PR.

**- How I Did It**
- Added root Home view component. 
- Added Hero component
- Added Marketing component
- Added marketing Point component

**- How to Test/Verify It**
- Pull branch
- launch dev server
- check new component on as many views as possible

**- An Image to Make Us Giggle (not mandatory but encouraged)**
![image](https://media.giphy.com/media/PI3AlEUhphmjC/giphy.gif)
